utils/field.js

```diff
@@ -183,7 +183,7 @@ const generateFakeDataForField = (field, faker, changelogDate, type = 'csv') =>
   return data;
 };
 
-function derivedProperties(field) {
+function _derivedProperties(field) {
```

Comment:
Why the function rename?

Reply:
It's a private one: I thought it was a good idea

Reply:
We should start to document our public/private api:
```
/**
 * @private
 **
```


But in this case, if function is exported for this use case, it should be public (blueprints may need it), since it's not exported it is effectively private and no need to document.